From d272c5d935b2ff8acc00f16317f1f960b02ed3a1 Mon Sep 17 00:00:00 2001
From: Sam James <sam@gentoo.org>
Date: Thu, 7 Sep 2023 21:23:51 +0100
Subject: [PATCH] topology: add include for ENABLE_NLS on musl

Needed for setlocale().

(After rebasing to pick up 8c229270f6bae83b705a03714c46067a7aa57b02, just
move it to be guarded as the include now exists.)

Closes: https://github.com/alsa-project/alsa-utils/pull/234
Signed-off-by: Sam James <sam@gentoo.org>
Signed-off-by: Jaroslav Kysela <perex@perex.cz>
---
 topology/topology.c | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/topology/topology.c b/topology/topology.c
index 5d03a8b133b8..98b000aa5031 100644
--- a/topology/topology.c
+++ b/topology/topology.c
@@ -32,11 +32,13 @@
 #include <sys/stat.h>
 #include <getopt.h>
 #include <assert.h>
-#include <locale.h>
 
 #include <alsa/asoundlib.h>
 #include <alsa/topology.h>
 #include "gettext.h"
+#ifdef ENABLE_NLS
+#include <locale.h>
+#endif
 #include "version.h"
 #include "topology.h"
 
-- 
2.35.3

